spring-kafka/src/main/java/org/springframework/kafka/core/KafkaOperations.java

```diff
@@ -64,6 +64,16 @@
 	ListenableFuture<SendResult<K, V>> sendDefault(int partition, K key, V data);
 
 	/**
+	 * Send the data to the default topic with the provided key and partition.
+	 * @param partition the partition.
+	 * @param timestamp the timestamp of the record.
+	 * @param key the key.
+	 * @param data the data.
+	 * @return a Future for the {@link SendResult}.
+	 */
+	ListenableFuture<SendResult<K, V>> sendDefault(int partition, Long timestamp, K key, V data);
```

Comment:
Yes, got a little confused here actually @artembilan , looks like the ProducerRecord uses a Long type and the consumer the primitive long. Since we were dealing with `send()` and that creates a ProducerRecord I actually kept the wrapper type of `Long`. Missed the javadoc, will add that in.